spark-operator/src/main/java/org/apache/spark/k8s/operator/config/SparkOperatorConf.java

```diff
@@ -0,0 +1,467 @@
+      ConfigOption.<Boolean>builder()
+          .key("spark.kubernetes.operator.metrics.clientMetricsEnabled")
+          .defaultValue(true)
+          .description(
+              "Enable KubernetesClient metrics for measuring the HTTP traffic to "
+                  + "the Kubernetes API Server. Since the metrics is collected "
+                  + "via Okhttp interceptors, can be disabled when opt in "
+                  + "customized interceptors.")
+          .build();
+
+  public static final ConfigOption<Boolean> KubernetesClientMetricsGroupByResponseCodeGroupEnabled =
+      ConfigOption.<Boolean>builder()
+          .key("spark.kubernetes.operator.metrics.clientMetricsGroupByResponseCodeEnabled")
+          .description(
+              "When enabled, additional metrics group by http response code group(1xx, "
+                  + "2xx, 3xx, 4xx, 5xx) received from API server will be added. Users "
+                  + "can disable it when their monitoring system can combine lower level "
+                  + "kubernetes.client.http.response.<3-digit-response-code> metrics.")
+          .defaultValue(true)
+          .build();
+
+  public static final ConfigOption<Integer> OperatorMetricsPort =
+      ConfigOption.<Integer>builder()
+          .key("spark.kubernetes.operator.metrics.port")
+          .defaultValue(19090)
+          .description("The port used for checking metrics")
+          .typeParameterClass(Integer.class)
+          .enableDynamicOverride(false)
+          .build();
+
+  public static final ConfigOption<Integer> OperatorProbePort =
+      ConfigOption.<Integer>builder()
+          .key("spark.kubernetes.operator.health.probePort")
+          .defaultValue(18080)
+          .description("The port used for health/readiness check probe status.")
+          .typeParameterClass(Integer.class)
+          .enableDynamicOverride(false)
+          .build();
+
+  public static final ConfigOption<Integer> SentinelExecutorServicePoolSize =
+      ConfigOption.<Integer>builder()
+          .key("spark.kubernetes.operator.health.sentinelExecutorPoolSize")
+          .description(
+              "Size of executor service in Sentinel Managers to check the health "
+                  + "of sentinel resources.")
+          .defaultValue(3)
+          .enableDynamicOverride(false)
+          .typeParameterClass(Integer.class)
+          .build();
+
+  public static final ConfigOption<Long> SENTINEL_RESOURCE_RECONCILIATION_DELAY =
+      ConfigOption.<Long>builder()
+          .key("spark.kubernetes.operator.health.sentinelResourceReconciliationDelaySeconds")
+          .defaultValue(60L)
+          .description(
+              "Allowed max time(seconds) between spec update and reconciliation "
+                  + "for sentinel resources.")
+          .enableDynamicOverride(true)
+          .typeParameterClass(Long.class)
+          .build();
+
+  public static final ConfigOption<Boolean> LEADER_ELECTION_ENABLED =
```

Comment:
Yes, this is the one I asked here.
- https://github.com/apache/spark-kubernetes-operator/pull/12/files#r1671269367